app/controllers/admin/help_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Admin::HelpController < ApplicationController
```

Comment:
Should this be `FAQsController`?

Reply:
I was going to change the model name to FAQ, so FAQsController sounds good.

Reply:
Yes it should be. I'll move the code in the help action to the index action in the FAQsController.

Reply:
I'm going to remove this controller and move the contents of the help action to the index action in the articles/questions controller.